src/main/java/nl/nn/testtool/Checkpoint.java

```diff
@@ -104,6 +106,14 @@ public Checkpoint(Report report, String threadName, String sourceClassName,	Stri
 		this.level = level;
 	}
 
+	public Span getSpan() {
```

Comment:
As this is a transient var I think you have to add @Transient and @JsonIgnore, see https://github.com/wearefrank/ladybug/blob/3440438628668b053ffe125e8cca0cdc6a95d630/src/main/java/nl/nn/testtool/Report.java#L79 The @JsonIgnore isn't documented I see but it is added to on most transient fields: https://github.com/wearefrank/ladybug/blob/3440438628668b053ffe125e8cca0cdc6a95d630/src/main/java/nl/nn/testtool/Report.java#L131

Reply:
Added the change